packages/app-desktop/gui/utils/NoteListUtils.test.ts

```diff
@@ -0,0 +1,120 @@
+import NoteListUtils from './NoteListUtils';
+import KeymapService from '@joplin/lib/services/KeymapService';
+import menuCommandNames from '../menuCommandNames';
+import { MenuItem as MenuItemType } from '@joplin/lib/services/commands/MenuUtils';
+import initializeCommandService from '../../utils/initializeCommandService';
+import { createAppDefaultWindowState } from '../../app.reducer';
+
+jest.mock('../../services/bridge', () => ({
+	__esModule: true,
+	default: () => ({
+		MenuItem: class MenuItem {
+			public value: MenuItemType;
+			public constructor(value: MenuItemType) {
+				this.value = value;
+			}
+		},
+		Menu: class MockMenu {
+			public items: MenuItemType[] = [];
+			public append(item: MenuItemType) {
+				this.items.push(item);
```

Comment:
CI seems to be failing due to the different keyboard shortcuts on MacOS.

Here are two ideas for resolving this:
1. Keep track of command IDs instead of all properties for each shortcut:
  ```suggestion
			  public itemKeys: string[] = [];
			  public append(item: MenuItemType) {
				  this.itemKeys.push(item.value.id);
  ```
2. Only check for the presence/absence of certain items. For example,
   ```
   expect(menu.items.map(item => item.value.id)).toContain('setTags');
   expect(menu.items.map(item => item.value.id)).not.toContain('delete');
   ```

Thank you for working on this!

Reply:
I think the only issue with keeping track of id is that not every MenuItem has it. Maybe it is a better approach anyways! Thanks for the help

Reply:
I end up a mix of both.

Since not every MenuItem has the same property I added some logic to decide which identifier to use. What do you think? Thanks again for the review!